src/main/java/net/mcreator/ui/component/util/SpinnerCellEditor.java

```diff
@@ -65,6 +66,8 @@ public Component getTableCellEditorComponent(JTable table, Object value, boolean
 		} else {
 			initialValSet = false;
 		}
+		if (eo instanceof MouseEvent event)

```

Comment:
Why is this change added and were consequences of the change tested in all other UI locations where this editor is used?

Reply:
This was made to allow selecting table cell without starting an edit, mostly needed for check box editors as without this part any click on a cell with it would toggle the contained check box. However, due to the main introduced component being transformed back to an entry list, these changes no longer deserve to be here.